README.md

```diff
@@ -129,7 +136,22 @@ entities (no need to draw an ERD):
 
 Your answer:
 ```
-Replace this with your answer
+An ERD is an Entity Relationship Diagram. It tells you how the different data in databases relate to each other.
+Genie
+-has_one :master
```

Comment:
while this is semantic, would make sure to match the name of the class you are associating with, i.e. `has_one :person`